### PR TITLE
Archive open API specs

### PR DESCRIPTION
**Pull request by ls-yannick**
Uploads the Open API spec fils to s3 so that we can later push them to the docs repo automatically.

(The first commit is formatting changes, and the last one is unrelated documentation. For easy review I suggest you mostly look at the second one :) )

### CODE REVIEW

**Review by jerome-leanspace — APPROVED** [review]
Assuming it has all the required permissions, it LGTM

**Comment by ls-yannick** [comment]
Actually it probably doesn't :D I still have to test this, should have put it as a draft.

**Review by coutinol — APPROVED** [review]

File: .github/workflows/sonar-nexus-deploy.yml [hunk 1]
```diff
@@ -98,8 +98,8 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          role-to-assume: ${{ format('arn:aws:iam::{0}:role/GithubActionsS3Access', secrets.ACCOUNT_ID) }}
-          aws-region: ${{ secrets.AWS_REGION }}
+          role-to-assume: arn:aws:iam::603457443348:role/GithubActionsS3Access
```
Comment (anmau19) [inline .github/workflows/sonar-nexus-deploy.yml:101]: Why do we want to hardcode the account ID? The repo is public so maybe we should avoid that

Comment (ls-yannick) [reply .github/workflows/sonar-nexus-deploy.yml]: It was for testing if the secrets were failing :)

**Comment by ls-yannick** [comment]
This appears to be working now https://s3.console.aws.amazon.com/s3/buckets/leanspace-github-build-artifacts?region=eu-central-1&prefix=activities-repository/0.1.81_1_f8ce8d1/&showversions=false so I will merge it and update the v1 tag to point to this version, unless there is a blocker :)

**Pull request merged**